### PR TITLE
Change parameters order for assertEquals in tests: JavaFutureTests and LeaseProviderTest

### PR DESCRIPTION
**Pull request by Captain1653**
assertEquals have signature is:
`assertEquals(expectedValue,actualValue)` [documentation](https://javadoc.io/doc/junit/junit/4.13/org/junit/Assert.html) , but there are some tests: JavaFutureTests and LeaseProviderTest with wrong parameters order `assertEquals(actualValue,expectedValue)` . 
I think it can be fixed.

Kind regards

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

File: akka-actor-tests/src/test/java/akka/dispatch/JavaFutureTests.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public String apply(String s) {
       }
     }, system.dispatcher());
 
-    assertEquals("Hello World", Await.result(f2, timeout));
+    assertEquals("Hello World", Await.result(f2, fiveSecondsTimeout));
```
Comment (raboof) [inline akka-actor-tests/src/test/java/akka/dispatch/JavaFutureTests.java:54]: I'm not sure this is helpful - it makes the assertions (somewhat) harder to read and it's not really relevant how long the timeout is...

**Review by raboof — COMMENTED** [review]
Thanks for fixing the parameter order!

I'm not sure I think the change from `timeout` to `fiveSecondsTimeout` is an improvement though.

**Review by raboof — APPROVED** [review]
:+1: thanks!

**Review by johanandren — APPROVED** [review]
LGTM, thanks

**Pull request merged**